hotel_reservation_summary_month/wizard/hotel_reservation_summary.py

```diff
@@ -0,0 +1,57 @@
+# Copyright 2020 Coop IT Easy SCRL fs
+#   Robin Keunen <robin@coopiteasy.be>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+
+from datetime import date
+
+from odoo import api, models
+from odoo.fields import Date
+
+
+class RoomReservationSummary(models.TransientModel):
+    _name = "hotel.room.reservation.summary.month"
+    _inherit = "hotel.room.reservation.summary"
+    _form_view = (
+        "hotel_reservation_summary_month"
+        ".hotel_room_reservation_summary_month_form_view"
+    )
+
+    @api.multi
+    @api.depends("date_from")
+    def _compute_name(self):
+        for summary in self:
+            month = Date.from_string(summary.date_from).strftime("%B")
+            summary.name = month
+
+    @api.model
+    def action_generate_summary(self):
+        first = date.today().replace(day=1)
+        next_month = (first.month + 1) % 12
```

Comment:
When accessing the page of december, I received the error
```
File "/home/odoo11/oca/vertical-hotel/hotel_reservation_summary_month/wizard/hotel_reservation_summary.py", line 45, in next_month_action
    date_to = date_from.replace(month=(date_from.month + 1) % 12)
ValueError: month must be in 1..12
````
I think the quick fix is
`next_month = first.month % 12 + 1`

Reply:
yep, you're right:

```
>>> for i in range(1, 13): print(i, i % 12 + 1)
...
1 2
2 3
3 4
4 5
5 6
6 7
7 8
8 9
9 10
10 11
11 12
12 1
```